requirements.txt

```diff
@@ -1,10 +1,10 @@
-django>=1.11,<2.2
+django>=1.11,<2.3
```

Comment:
change to `django>=1.11,<3.0`

Reply:
That will install 2.8 right ? Okay :smile: